packages/next/src/server/base-server.ts

```diff
@@ -519,7 +519,23 @@ export default abstract class Server<ServerOptions extends Options = Options> {
   ): Promise<void> {
     return getTracer().trace(
       BaseServerSpan.handleRequest,
-      async () => await this.handleRequestImpl(req, res, parsedUrl)
+      {
+        tracerName: [req.method, req.url].join(' '),
```

Comment:
Before `tracerName` was introduced, the span name was always this "span identifier" name, as defined in the constants file. Having more meaningful names for the spans is a good thing -- but this information about the "kind of span" that you're dealing with is also useful.

Would it make sense to have this identifier always be sent as an attribute, `next.span` or something like that?

Reply:
Great point :+1:

Reply:
Done.

Reply:
Thank you! I can't find where the change was made though 🤔

Reply:
Whoops, wrong branch. Re-added it here.
https://github.com/vercel/next.js/pull/47066/files#r1135667923